src/constants.py

```diff
@@ -13,3 +13,18 @@
 SDAI_TOKEN_ADDRESS = Web3.to_checksum_address(
     "0x83F20F44975D03b1b09e64809B757c47f942BEeA"
 )
+
+# Time limit after which Coingecko Token List is re-fetched (in seconds)
+COINGECKO_TOKEN_LIST_RELOAD_TIME = 86400
+
+# Time in seconds of 45 hours. Time limit after which 5-minute prices become unavailable.
+COINGECKO_TIME_LIMIT = 162000
+
+# Buffer time interval to allow 5-minutely Coingecko prices to be fetched
+COINGECKO_BUFFER_TIME = 600
+
+# Dune query for fetching prices is set to LIMIT 1, i.e. it will return a single price
+DUNE_PRICE_QUERY_ID = 3935228
+
+# Dune Query 3935228 uses an end_timestamp to limit results
```

Comment:
Comment seems to be referring to something else

Reply:
It's correct. This means that to avoid dune from returning all results from start_timestamp till now, the query uses end_timstamp to limit results, by adding a fixed time as buffer time. (currently 100 secs). 
I can mention it as buffer time for it to be clear.

Reply:
oh i see. yeah the name is not very clear but i guess it's fine